src/portal/lib/src/repository-gridview/repository-gridview.component.ts

```diff
@@ -169,12 +171,14 @@ export class RepositoryGridviewComponent implements OnChanges, OnInit {
                     } else {
                         this.clrLoad(st);
                     }
+                }, error => {
+                    this.errorHandler.error(error);
                 });
             }
         }
     }
 
-    delOperate(repo: RepositoryItem) {
+    delOperate(repo: RepositoryItem): Observable<any> | null {
```

Comment:
when `null` returned, is the observable pipeline still work?

Reply:
it return null when forkin code run  in error,so it won't affect the results.

Reply:
> it return null when forkin code run in error,so it won't affect the results.

We didn't write a return value when we used promise, but this function actually has a return value promise object or undefined. Now I change undefined to a null pointer.